### PR TITLE
Check for code review

### PR DESCRIPTION
**Pull request by Kgaur624**
Someone check my code if its possible 
while i am posting the tweet its showing server error 
before next meeting

### CODE REVIEW

File: Droppwiz/src/main/java/com/resource/FetchTweets.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        arr=new String[size];
+        System.out.println("Showing home timeline.");
+        int i=0;
+        for (Status status1 : statuses)
+        {
+            arr[i]=status1.getText();
+            i++;
+        }
+        if (size == 0){
+            arr[0] = "No Tweet Found ";
```
Comment (SupriyaPamulapati) [inline Droppwiz/src/main/java/com/resource/FetchTweets.java:32]: you can add error 400

**Review by SupriyaPamulapati — COMMENTED** [review]
Please do address comments

**Review by SupriyaPamulapati — COMMENTED** [review]
Please address the comments

File: Droppwiz/src/main/java/com/resource/FetchTweets.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                .setOAuthConsumerKey("jUw2qaXkcqirbpR0h76dGQPEb")
+                .setOAuthConsumerSecret("KlrnGLxD3QfQatAidusemOX8T7OL072Vi2HJppCMrxGL2svkcY")
+                .setOAuthAccessToken("1450680649031962626-FlE4DbG86871LzJ6dVrA77aMjR47cy")
+                .setOAuthAccessTokenSecret("gEIxm1eWf6ON47WszmZMzm6v9rtzWTo4LGJtIdygsJFeU");
+        TwitterFactory tf = new TwitterFactory(cb.build());
+        Twitter twitter = tf.getInstance();
+        List<Status> statuses = twitter.getHomeTimeline();
+       size = statuses.size();
+        arr=new String[size];
+        System.out.println("Showing home timeline.");
```
[the remark this answers is not in the document]
Comment (Kgaur624) [reply Droppwiz/src/main/java/com/resource/FetchTweets.java]: i have done

File: RestApi/src/main/java/com/resource/Mainn.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/api/1.0/twitter")
+public class Main {
+    @POST
+    @Path("/tweet")
```
[the remark this answers is not in the document]
Comment (Kgaur624) [reply RestApi/src/main/java/com/resource/Mainn.java:19]: i have done

File: Droppwiz/src/main/java/com/resource/FetchTweets.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    class FetchTweets {
+    public  static  String[] latestTweet() throws TwitterException {
+        int size;
+        String arr[] ;
+        ConfigurationBuilder cb = new ConfigurationBuilder();
+        cb.setDebugEnabled(true)
+                .setOAuthConsumerKey("jUw2qaXkcqirbpR0h76dGQPEb")
+                .setOAuthConsumerSecret("KlrnGLxD3QfQatAidusemOX8T7OL072Vi2HJppCMrxGL2svkcY")
+                .setOAuthAccessToken("1450680649031962626-FlE4DbG86871LzJ6dVrA77aMjR47cy")
+                .setOAuthAccessTokenSecret("gEIxm1eWf6ON47WszmZMzm6v9rtzWTo4LGJtIdygsJFeU");
```
[the remark this answers is not in the document]
Comment (anasuya1117) [reply Droppwiz/src/main/java/com/resource/FetchTweets.java]: this is a security loophole, should be removed and moved to properties file

**Pull request closed without merging**